samples/Demo/speedometer-consumer/src/main.rs

```diff
@@ -164,6 +176,74 @@ async fn receive_vehicle_speed_updates(
     Ok(sub_handle)
 }
 
+fn running_led(panel: &mut led_driver::ws2811_t, led_color: u32, delay_ms: u64, max_count: i32)
```

Comment:
all of this new code needs to go into `#[cfg(target_arch = "aarch64")]` block so it is not compiled for x86, compilation for x86 fails because `led_driver` is not compiled in